### DISCUSSION TITLE
Change repo url

### DISCUSSION THREAD

**Discussion by Qadyan-dev** [codemagic-ci-cd/codemagic-docs#969] [Q&A] [answered]
I would like to change repository could you help

**Comment by K-Codemagic** [comment]
Hi, can you please elaborate? Have you migrated the repository somewhere else?

**Comment by adm-merchandi** [comment]
@K-Codemagic Hi! 

We moved our repo from one org to another and we would like to keep our current config and environment variables so we transfer it through GitHub with the hopes that the project id would help code-magic to know the new URL, but when we try to update the URL it says error and now we cannot deploy our app

We just want to be able to update de URL, currently, we can see the string but we can't update it

![Screenshot 2023-01-12 at 19 50 11](https://user-images.githubusercontent.com/106703015/212233111-39c5866e-57db-4a74-9a98-fa93bbc1dd59.png)

**Reply by K-Codemagic** [reply]
Hi, can you add the new app with the new URL and share the old and the new app id, so I can change it in the backend, that would keep your configuration intact.

**Reply by adm-merchandi** [reply]
@K-Codemagic 

 Hi thank u!

This is the old app: https://codemagic.io/app/63056dbb6eb167a5cdd39b1b
This is the new app: https://codemagic.io/app/63c6ffbfe2a44e3553323fea

Not sure where to get the ID but I think is in the url?
Can u move the environment variables to our team environment? in that way we could share it for all our apps

**Reply by K-Codemagic** [reply]
Hi, I have updated the URL of your old app. You can please check and confirm?

**Reply by adm-merchandi** [reply]
@K-Codemagic yes is working thank u!

**Comment by K-Codemagic** [comment, marked as answer]
Closing this as the issue is resolved.
